functions/activate-batch-auto-background.js

```diff
@@ -78,6 +88,28 @@ async function handle(_, db, contract) {
     let result = await db.query('batches', 'batch', batch)
     const existingAddresses = result.Items
 
+    let addressBadluckCount = {};
+    for(let i = 0; i < existingAddresses.length; i+=MAX_CONCURRENCY) {
+      await Promise.all(existingAddresses.slice(i, i+MAX_CONCURRENCY).map(async a => {
+        let account = {
+          address: a.address,
+          inventory: '[]',
+          fragments: 0,
+          badLuckCount: 0,
+          discord: "",
+        }
+    
+        const fetchedAccount = (await db.get('accounts',{ address: a.address })).Item
+    
+        if(fetchedAccount !== undefined) {
+          account = fetchedAccount
```

Comment:
I'll update this line to `account = { ...account, ...fetchedAccount }` and it should account for existing accounts that don't have a bad luck counter